components/browser/engine-gecko-beta/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngineSession.kt

```diff
@@ -132,6 +134,10 @@ class GeckoEngineSession(
      * See [EngineSession.enableTrackingProtection]
      */
     override fun enableTrackingProtection(policy: TrackingProtectionPolicy) {
+        if (policy == TrackingProtectionPolicy.none()) {
+            return
+        }
```

Comment:
Is this something we strictly need?

I wonder how that would work in Focus where we can have tracking protection on but all categories disabled.

Reply:
Not strictly, no. I removed it.